src/observer/logic.py

```diff
@@ -7,3 +7,9 @@ def known_articles():
         .values_list('msid', flat=True) \
         .order_by('-msid') \
         .distinct()
+
+def verified_subjects(string_list):
+    "given a list of subject slugs, returns only those present in the database"
+    return models.Subject.objects \
+        .filter(name__in=string_list) \
+        .values_list('name', flat=True) # ['foo', 'bar', 'baz']
```

Comment:
I guess these are populated from the articles that have been notified on the bus?
What happens for a new subject for which no article has been ingested or published yet?

Reply:
(it would be easy to listen to subjects on the bus too, if needed)

Reply:
if no article uses a subject, does that subject matter? (pun, sorry)

from the pov of Observer, it's no. Observer is article centred. If we ever expand it's scope to non-article data this might come up again then

Reply:
I feel @davidcmoulton should be a witness to my glorious pun. it may be my first and only ever.

Reply:
As an observer I've also been subjected to it :)